YoCode/YoCode.csproj

```diff
@@ -16,6 +16,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="2.1.1" />
     <PackageReference Include="MSBump" Version="2.3.2" />
+    <PackageReference Include="Nerdbank.GitVersioning" Version="2.1.65" />
```

Comment:
Same comment as I made for the test project,
Did you mean to add this? I don't think it's necessary. It's certainly not relevant to this PR. We should remove the reference to MSBump too since we're not going to use it.